index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/5116129a4d.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+   <section class="header">
+    <div class="name">Daniel</div>
+    <div class="hamburger"><i class="fas fa-hamburger"></i></div>
+   </section>
+
+   <section class="headline">
+    <h1 class="intro">Hey There. I'm Daniel</h1>
+   <p class="developer">I'm a Software developer</p>
+   <p class="abilities">I can help you build a product , feature or website Look through some of my work and experience! 
+    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="icons">
+        <a href="" class="github"><i class="fa-brands fa-github"></i></a>
+        <a href="" class="linkedin"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        <a href=""><i class="fab fa-angellist"></i></a>
+        <a href="" class="twitter"><i class="fa-brands fa-twitter"></i></a>
+        <a href="" class="facebook"><i class="fa-brands fa-facebook-f"></i></a>
+    </div>
```

Comment:
- [ ] For a list of items, always use the unordered list or `ul` tag and each list item can then be wrapped in a `li`. The list of icons thus should be wrapped in a `ul` tag instead of a `div` to maintain its semantic meaning. Kindly alter this code to use an unordered list tag instead.